### DISCUSSION TITLE
Standard CreateResourceAction modal -> mutateFormDataUsing()

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#7945] [Help]
How to alter $data inside the standard create resource button, when its in modal? I got an mutateFormDataBeforeCreate(array $data) inside my CreateResource Class, and its works when i use it as a page, but not in modal.

**Comment by pboivin** [comment]
Maybe this can help : https://filamentphp.com/docs/3.x/actions/prebuilt-actions/create#lifecycle-hooks

**Comment by RPotkowaVegasystems** [comment]
Yes, i actually created a HeaderAction with a new button with the CreateAction mehthod incl. mutating.. And this works as well. But the Problem is, when i set "canCreate() to false" to disable the standart Button in the right top corner, my new create Button disappers also.
